### ISSUE TITLE
Classes?

### ISSUE THREAD

**Issue by ghost** [andrewjcourt/Dealer#1]
This looks like the sort of thing that could use classes really effectively. See https://www.tutorialspoint.com/python/python_classes_objects.htm

**Comment by andrewjcourt** [comment]
Yep. Unfortunately I haven’t reached the chapter on classes in my python textbook yet.
